src/strategies/java-yoshi-mono-repo.ts

```diff
@@ -187,10 +187,10 @@ export class JavaYoshiMonoRepo extends Java {
         }),
       });
 
-      // The artifact map maps from directory paths in repo to artifact names on
-      // Maven, e.g, java-secretmanager to com.google.cloud/google-cloud-secretmanager.
-      const artifactMap = await this.getArtifactMap('artifact-map.json');
-      if (artifactMap && options.commits) {
+      // Bail early if the repository has no root changelog.json.
+      // This file is used to opt into machine readable commits.
+      const hasChangelogJson = !!(await this.hasChangelogJson());
```

Comment:
Is `!!` necessary, given that it is already returning a `Boolean`?

Reply:
whoops, my bad held over code.